channels/es.m3u

```diff
@@ -315,8 +315,6 @@ http://5.189.181.203:1935/gcm4/gcm4/chunklist.m3u8
 http://ventdelnord.tv:8080/girona/directe.m3u8?fluxustv.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://media.tvalacarta.info/canales/256x256/tvmelilla.png" group-title="",Televisión Melilla
 https://melillatv-hls.flumotion.com/melillatv/melillatv_all/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/OKxWzyO.png" group-title="",Tenerife 2030
-http://livestream.iter.es:9876/tdt/tdt.m3u8
```

Comment:
Link works (Tested in VLC)

Reply:
Strange, for me the channel is still not working. Here for example what gives out VLC player:
<img width="734" alt="Screenshot 2019-10-12 at 20 04 55" src="https://user-images.githubusercontent.com/7253922/66705162-eef63400-ed2b-11e9-9a68-827be728bbdb.png">

Reply:
![afbeelding](https://user-images.githubusercontent.com/25525836/66706513-e73a8d80-ed33-11e9-899b-23da2e5f457f.png)
- When i test this one now it is not playing in VLC , this morning it did.
- Right Now i did also test on streamtest.in ( see image) there it works , and also in kodi it plays. 
It is not a dead link , it is a VLC problem but i encounter more and more links with this issue.

Reply:
You could try this one : http://livestream.iter.es:9876/tdt/tdt_720.m3u8
This one i pulled directly from the website -> http://www.canaltenerifetv.com/channels/tenerife-tv
And try if you can play it in your browser

Reply:
http://livestream.iter.es:9876/tdt/tdt_720.m3u8 - doesn't work either
And the site also does not open:
<img width="527" alt="Screenshot 2019-10-13 at 04 43 39" src="https://user-images.githubusercontent.com/7253922/66709789-0ce78700-ed74-11e9-9d16-9355f02a0bc1.png">